xclim/ensembles.py

```diff
@@ -718,11 +718,11 @@ def plot_rsqprofile(fig_data):
 
     Examples
     --------
-    >>> import xarray as xr  # doctest: +SKIP
-    >>> from xclim.ensembles import kmeans_reduce_ensemble, plot_rsqprofile  # doctest: +SKIP
-    >>> crit = xr.open_dataset(path_to_ensemble_file).data  # doctest: +SKIP
-    >>> ids, cluster, fig_data = kmeans_reduce_ensemble(data=crit, method={'rsq_cutoff':0.9}, random_state=42)  # doctest: +SKIP
-    >>> plot_rsqprofile(fig_data)  # doctest: +SKIP
+    >>> from xclim.ensembles import kmeans_reduce_ensemble, plot_rsqprofile
+    >>> is_matplotlib_installed()
+    >>> crit = xr.open_dataset(path_to_ensemble_file).data
+    >>> ids, cluster, fig_data = kmeans_reduce_ensemble(data=crit, method={'rsq_cutoff':0.9}, random_state=42)
+    >>> plot_rsqprofile(fig_data)
     """
```

Comment:
```suggestion
    """
        if not MPL_INSTALLED:
        logging.error("Matplotlib is not installed. Unable to execute plotting functions.")
        raise ImportError("matplotlib")

```
Is this a terrible idea?

Reply:
bah... Not a bad idea, but isn't the name "plt_rsqprofile" already quite explicit on the fact that it does plot? (And thus needs matplotlib)?

Reply:
I suppose so. Nothing prevents this from being called as is. If MPL isn't there, I suppose the errors from python ("plt is not defined" or something like that) would be enough?

Reply:
Yes, seems overkill. Also, maybe we should avoid creating graphics in the doctests. If it's important to show these graphics, I would place them in the notebook tutorials instead.

Reply:
True! We could have a notebook on all "ensembles" things, including kmeans and kkz selection.

Reply:
See #522